library/headers.inc.php

```diff
@@ -70,6 +70,11 @@ function call_required_libraries($library_array){
         <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery-datetimepicker/jquery.datetimepicker.full.min.js"></script>
     <?php
     }
+    if (in_array("jquery-ui",$library_array)){   ?>
+        <link rel="stylesheet" href="<?php echo $GLOBALS['css_path']; ?>jquery-ui-1-12-1/jquery-ui.css" media="screen" />
+        <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery-ui-1-12-1/jquery-ui.js"></script>
+    <?php
+    }
```

Comment:
Thanks for adding this.
Maybe we can pop out some more tasks/issues to update the implementation of jquery-ui elsewhere now.

Reply:
This library is really awesome!